Automattic/jetpack
# Remove duplicate views: Update experiment name

Part of https://github.com/Automattic/wp-calypso/issues/97398

## Proposed changes:

We used a fake experiment (22124-explat-experiment) in https://github.com/Automattic/jetpack/pull/40471 so we could start the "Remove duplicate views" project without being blocked by the lack of the experiment properly designed in ExPlat.

We now have a real experiment (22129-explat-experiment), so this PR replaces the previous experiment name with the right one.

### Other information:


## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
- Go to the final experiment page: 22129-explat-experiment
- Assign yourself to the `treatment` variation using the bookmarklets
- Visit any of these screens:
  - Posts > All Posts
  - Posts > Tags
  - Posts > Categories
  - Portfolio > All Projects
  - Testimonials > All Testimonials
- Make sure the WP Admin view is enforced for all of them
